homeassistant/components/google_mail/const.py

```diff
@@ -0,0 +1,19 @@
+"""Constants for Google Mail integration."""
+from __future__ import annotations
+
+ATTR_ENABLED = "enabled"
+ATTR_END = "end"
+ATTR_MESSAGE = "message"
+ATTR_PLAIN_TEXT = "plain_text"
+ATTR_RESTRICT_CONTACTS = "restrict_contacts"
+ATTR_RESTRICT_DOMAIN = "restrict_domain"
+ATTR_START = "start"
+ATTR_TITLE = "title"
+
+DOMAIN = "google_mail"
+
+MANUFACTURER = "Google, Inc."
+DEFAULT_ACCESS = [
+    "https://mail.google.com/",
```

Comment:
I don't think full access to the mailbox is needed so drop?

Reply:
I'm think readonly and compose will be good?

Reply:
Or I can just omit it for now since it is not needed for Vacation settings.

Reply:
I'm definitely a fan of requesting just what is needed. However, if there are followup PRs planned and the primary function of this is to support reading and composing emails and just getting vacation responders is less of the focus, then i get that just having the broader access makes sense (e.g. many ui options etc would not worthwhile if its not the primary point). I think i'm just messing what the larger plan is here vs what is possible, but we have no intent to actually do.

Reply:
To make this integration have a bit more meaning on release, I've just added the email part. Would be kind of silly to release without the features for it's namesake.